package.json

```diff
@@ -948,7 +948,7 @@
         "mkdirp": "0.5.1",
         "mocha": "3.1.2",
         "nyc": "^11.4.1",
-        "oni-release-downloader": "0.0.8",
```

Comment:
Looks like this is the issue!
`0.0.8` doesn't have any code to deal with x64 so all windows builds are `win32`.
Could that cause the issues we've seen?

I was checked the individual packages for this, but not here apparently.

Reply:
Ah excellent! 👍  Makes sense now - the `peerDependencies` in the other packages just specify their 'requirement' for a package, and this is where we actually acquire the dependency to use. It's confusing, there's probably a better way to handle this..  Glad we got it going, thanks @CrossR ! 🎉